src/pages/homePage.js

```diff
@@ -1,19 +1,32 @@
-import { Link } from 'react-router-dom';
+import { Image, Layout, Breadcrumb } from 'antd'
+import { Link } from 'react-router-dom'
+import { routes } from '../config/routes'
+import logo from '../styles/images/logo.png'
+import '../styles/homeForm.scss'
 
 export const HomePage = () => {
-    return (
-        < >
-            <div><h1>THIS IS HOMEPAGE</h1></div>
-
-            <div>some information</div>  
-            
-            <div>
-                <p>please Login </p>
-                    <Link to='/login'>
-                        <p>to Login</p>
-                    </Link>
-            </div>
-
-        </>
-    )
+  const { Header, Content, Footer } = Layout
+  return (
+    <Layout>
+      <Header className="header">
+        <Image width={65} src={logo} />
+      </Header>
+      <Content className="site-layout">
+        <Breadcrumb className="breadcrumb">
+          <Breadcrumb.Item>
+            <Link to={routes.homepage}>Home</Link>
+          </Breadcrumb.Item>
+          <Breadcrumb.Item>
+            <Link to={routes.login}>Login</Link>
+          </Breadcrumb.Item>
+        </Breadcrumb>
```

Comment:
Breadcrumbs are used by programmers to show the whole navigation path to user and allow user to navigate back for 1, 2 or more steps. Pls refactor this with some another component or just div. also from my point of view it should be placed at header, not in Content component.